frontend/src/index.less

```diff
@@ -6,10 +6,6 @@
   background: #333;
   color: #fff;
   margin: 0;
-
-  &:before {
-    content: "$ "
-  }
```

Comment:
The point of having this in a `:before` is that the dollar sign is not copied when you select the whole block

Reply:
The pre-existing css is the strangest I've ever seen. What's up with all the `nth-child` selectors? Positional as opposed to named css? That's just an awful idea.
So I can't blame Matthew for not restoring this functionality for nix-env. He had to remove the `$` for the declarative installation methods, which are more important anyway.

Reply:
I know, it's awful. There's no need to stick to that pattern though: adding a `shell-commands` class outside the hierarchy would be fine.

Reply:
I am not familiar with elm enough to accomplish this. If you could make a PR that does it to my fork, I'll merge it and it'll be done. But I think the scope of this PR will then have been increased more than it needs to be.

Reply:
No problem! I pushed to your branch directly.